spark-bukkit/src/main/java/me/lucko/spark/bukkit/BukkitSparkPlugin.java

```diff
@@ -52,6 +52,9 @@
 import java.util.logging.Level;
 import java.util.stream.Stream;
 
+import java.util.logging.LogManager;
```

Comment:
Just added logging capabilities so we can get more detailed warnings and errors for debugging purposes